.github/ISSUE_TEMPLATE/feature.yml

```
@@ -0,0 +1,15 @@
+name: Feature
+description: 새로운 기능 추가 혹은 기존 기능 변경이 필요할 때 발행할 이슈.
+labels: ["enhancement"]
```

feature에 대한 이슈라면 라벨도 feat으로 하면 어떨까요?
```suggestion
labels: ["feat"]
```

---

넵, 그리고 라벨을 만든다면 feature가 좋지 않을까 싶은데 어떠신가요?
커밋로그는 짧을수록 좋으니까 feature를 feat으로 줄여서 작성하지만
라벨은 그럴 필요가 없으니까요.

---

그것도 좋긴 한데, 만약에 라벨명이 `conventional commit` 종류를 따르지 않는다면 아예 전부 한글로 적는 편이 나을 것 같아요.  `버그`, `기능 추가` 같은 식으로요.